portfolio/src/main/java/com/google/sps/data/ReviewerData.java

```diff
@@ -0,0 +1,18 @@
+package com.google.sps.data; 
+
+public final class ReviewerData {
```

Comment:
What does this class represent?

I am having trouble probably because "Data" is probably the most generic name you can find to describe any kind of "data" and thus doesn't help me understand what this class represents. At first sight, this is what comes into my mind when seeing this class: "this is a reviewer, who has a name and a comment."

Reply:
Changed it to "CommentDetails"